src/pygenesis/http_helper.py

```diff
@@ -83,14 +83,68 @@ def get_data_from_endpoint(endpoint: str, method: str, params: dict) -> str:
     )
 
     response = requests.get(url, params=params_, timeout=(5, 15))
-    response.encoding = "UTF-8"
 
+    # if the response requires starting a job, new params are generated and the job is started
+    try:
+        # test for job-relevant status code
+        response_status_code = response.json().get("Status").get("Code")
+
+        if response_status_code == 98:
+            logger.info(
+                "Die Daten sind zu groß um direkt geladen zu werden, es wird ein Job angestoßen."
+            )
+            # updating params to start a job
+            new_params = params.copy()
+            new_params.update({"job": "true"})
+
+            # starting a job
+            jobs_response = load_data(
+                endpoint=endpoint,
+                method=method,
+                params=new_params,
+                as_json=True,
+            )
+
+            return str(_jobs_job_id(jobs_response))
+    except json.decoder.JSONDecodeError:
+        pass
+
+    response.encoding = "UTF-8"
     _check_invalid_status_code(response.status_code)
     _check_invalid_destatis_status_code(response)
 
     return str(response.text)
 
 
+def _jobs_job_id(response) -> requests.Response:
+    """
+    Helper method which handles too large data requests and gives access to job id.
+
+    Args:
+        response (json): Response from endpoint request with job set equal true
+
+    Returns:
+        response: either failing response or response after starting a job
+    """
+    # check out job_id & inform user
+    s = response.get("Status").get("Content")
+    job_id = s.split(":")[1].strip()
+
+    # Notifying user about successfully starting a job and returning the response of said request
+    logger.info(
+        "Der Status des Jobs kann über den catalogue/jobs Endpunkt "
+        "mit dem Kriterium 'selection': '*%s' abgerufen werden",
+        job_id,
+    )
+    logger.info(
```

Comment:
siehe oben/ könnte man mMn in eine Nachricht packen, da die ID ja identisch ist/ keine neuen Informationen entstanden sind.